install-wrappers.sh

```diff
@@ -50,20 +50,20 @@ else
 fi
 cd $PREFIX/bin
 for arch in $ARCHS; do
-  for target_os in $TARGET_OSES; do
-    for exec in clang clang++ gcc g++ cc c99 c11 c++; do
-        ln -sf clang-target-wrapper$CTW_SUFFIX $arch-w64-$target_os-$exec$CTW_LINK_SUFFIX
+    for target_os in $TARGET_OSES; do
```

Comment:
Please squash this change into the original file that added the new indentation level

Reply:
Done